calibration/DOSTAM/CGINS-DOSTAM-00123__20191009.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+123,CC_calibration_date,1539043200000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
I think the date format in the vendor cal doc is day-month-yr so this should be 20190910 (updated vendor filename in Vault); milliseconds needs to be updated as well (make sure you do yr 2019...what you have currently is for 2018 plus the wrong day-month)